app/assets/javascripts/spree/backend/templates/available_locales.hbs.erb

```diff
@@ -0,0 +1,28 @@
+<td>
+  {{store.name}}
+  {{#if store.default}}
+    <span class="label label-default">default</span>
+  {{/if}}
+</td>
+<td>{{store.url}}</td>
+<td>
+  {{#if editing}}
+    <select id='available-locales-store-{{store.id}}' class='available-locales select2 fullwidth' name='store[preferred_available_locales][]' multiple='true'>
+      {{#each availableLocales}}
+        <option value="{{ this }}" {{#selected this ../store}}{{/selected}}>
+          {{#locale-presentation this ../availableLocalesPresentation}}{{/locale-presentation}}
+        </option>
+      {{/each}}
+    </select>
+  {{else}}
+    {{store-available-locales store availableLocales}}
+    {{#each store.preferences.available_locales}}
+      {{locale-presentation this}}
+    {{/each}}
+  {{/if}}
+</td>
+<td class="actions">
```

Comment:
can we use `{{#if editing}}` to conditionally display actions instead of hiding/showing with css?